junit-jupiter-api/src/main/java/org/junit/jupiter/api/TestFactory.java

```diff
@@ -34,6 +34,9 @@
  * will then be executed lazily, enabling dynamic and even non-deterministic
  * generation of test cases.
  *
+ * <p>Any Stream returned by a {@code @TestFactory} will be properly closed by calling {@code stream.close()},
+ * making it safe to use a resource, for example {@code Files.lines()}.
```

Comment:
I think it would read better if this line were to be replaced with something like,

> making it safe to use a resource **such as** {@code Files.lines()}.